azure/services/virtualmachines/client.go

```diff
@@ -29,19 +29,17 @@ import (
 	"sigs.k8s.io/cluster-api-provider-azure/util/tele"
 )
 
-type (
-	// AzureClient contains the Azure go-sdk Client.
-	AzureClient struct {
-		virtualmachines *armcompute.VirtualMachinesClient
-	}
+// AzureClient contains the Azure go-sdk Client.
```

Comment:
Was this diff intended? It looks like it's just changing how the type is wrapping the structs

Reply:
I think this may be left over from when I had the other interfaces,  I can change it back if it's more readable.